util/host.go

```diff
@@ -37,3 +41,18 @@ func EnsureHost(addr string) string {
 	}
 	return net.JoinHostPort(host, port)
 }
+
+// EnsureFullAddr check if addr is a host:port pair. It make sure to have both.
+func EnsureFullAddr(addr string) string {
```

Comment:
@tschottdorf Sorry, I do not get you point. You mean we should have only one constant 26257 in code?